projects/gradientci-v2/README.md

```diff
@@ -229,6 +229,10 @@ For some `step`s in your job GradientCI will automatically set environment varia
 * `PS_GRADIENT_CI_GIT_REPO_URL` the repository that received the trigger.
 
   Note that builds from forks will appear to come from the _target_ repository, not the fork.
+  
+## S3 Datasets
```

Comment:
Can you also update an example that runs an experiment to include a dataset using secrets?

Reply:
@bbatha good point, updated one of the examples let me know if this looks good 👍